### ISSUE TITLE
Design Intent - Accessibility Feedback - mhv-secure-messaging, Filter Redesign

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#91102]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the late feedback! I completely blanked on this Friday afternoon and over the weekend.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** My biggest point of concern here is having so many expandable/conditionally revealing chunks of content nested within each other: first the accordion, then the "Add filters +" element, then the conditonally-revealed dates for the custom date range. Thinking about screen reader users, I think it's a situation where someone could lose track of what's open, what's close, what's associated with what, etc.
   
   Thinking about it since the meeting, I think my recommendation here is to remove the "Add filters +" element and just make all of the filters visible when opening the accordion. It's two extra tab stops (assuming not doing custom dates), which is manageable for keyboard-only users. Given the other trade-offs I think it's the best option.
   
   I'll note that the custom date reveal doesn't align with [design system guidance on conditionally-revealed fields](https://design.va.gov/patterns/ask-users-for/a-single-response#conditionally-revealed-fields). That guidance was written with more traditional form flows in mind and I think deviating from it is fine in this context. I'm commenting on that here mainly so that I remember this is where I landed later on. 🙂 
- [ ] **Should:** I'll echo Erin on the summary text that describes what messages are shown. Rather than two lines of text ("[results] matches for..." and "Showing X to Y..."), it would be great to combine these into a single message, eg. "Showing X to Y of Z matches for [filter criteria]" or similar.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by underpaid1ntern** [comment]
CC: @emilytaylorux
